kwords.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
+
+def get_keywords(text):
+
+    payload = json.dumps({
+    "text": text,
+    "lang": "en",
+    "n": 10
+    })
+    headers = {
+    'Content-Type': 'application/json'
+    }
+
+    response = requests.request("POST", url, headers=headers, data=payload)
+
+    keywords, scores, relevant_keywords = response.json()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's no error handling for the HTTP request. If the request fails for any reason (e.g., the server is down, the response is not JSON), the program will crash. Consider adding some error handling here.
```diff
- response = requests.request("POST", url, headers=headers, data=payload)
- keywords, scores, relevant_keywords = response.json()
+ try:
+     response = requests.post(url, headers=headers, data=payload)
+     response.raise_for_status()
+ except requests.exceptions.RequestException as err:
+     print(f"HTTP Request failed: {err}")
+     return None
+ 
+ try:
+     keywords, scores, relevant_keywords = response.json()
+ except json.JSONDecodeError:
+     print("Failed to decode the response from server")
+     return None
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->